djangocms_frontend/contrib/icon/conf.py

```diff
@@ -26,11 +26,8 @@
     "foundation-icons",
     "elegant-icons",
     "feather-icons",
-    "happy-icons",
-    "icomoon",
```

Comment:
I'm hearing rumours that there is still something trying to load these.

Reply:
Any indications? They're still in the docs though...

Reply:
Well, I picked up this issue: django-cms/django-cms-quickstart#58 and traced the error back to this PR. Downgrading to `1.2.0` seems to resolve the bug. I have not gone deep into this yet, but I can still grep for files with the same name as those icon libraries: `/djangocms_frontend/contrib/icon/static/djangocms_frontend/icon/vendor/assets/icons-libraries/icomoon.json` (not sure if that's relevant at the moment).

Reply:
This might be related to my findings in https://github.com/django-cms/djangocms-frontend/issues/166#issuecomment-1857837518 from about a month ago, id est that some files still reference the old icon sets.

Reply:
@stefan6419846 It is! Thank you! The css files reference the removed fonts which breaks whitenoise. I have removed the css files a month ago. This needs to go through the release process now!

The json files are staying, so that people fine with the iconset's license can use them more easily.